config/file_store.yml

```diff
@@ -1,18 +1,14 @@
 development:
-  storage: local
-  # the secure option in this file is deprecated, use the ssl option in domain.yml instead
-  # secure: false
-  path_prefix: tmp/files
-
+#   storage: local
+#   # the secure option in this file is deprecated, use the ssl option in domain.yml instead
+#   # secure: false
+#   path_prefix: tmp/files
+  storage: s3
+  
```

Comment:
I don't want to change development to use S3 storage

Reply:
Oh sap! =) FYI: this allows allow for caching to work locally with Shards without having to use all the other RAILS_ENV=production stuff.

Why isn't it just using the cache_store.yml? Because canvas an for us WIP. FYI: Caching will need to be revisited to be fully working. This solution is not optimized for it on multiple shards but it looks reasonably strait forward.